corehq/sql_db/models.py

```diff
@@ -61,7 +61,7 @@ def partitioned_query(self, partition_value=None):
     def using(self, alias):
         return self.db_manager(hints={'using': alias})
 
-    def raw(self, raw_query, params=None):
+    def plproxy_read(self, raw_query, params=None):
         return RawQuerySet(raw_query, model=self.model, params=params, hints={'plproxy_read': True})
```

Comment:
Is the name change just to make it easier to read / more explicit?

Reply:
Yes though I'm not actually sure I like it. Initially I wanted to make it clear that this would always be routed to plproxy as a read only query. Perhaps a better name would be `plproxy_raw`?